### DISCUSSION TITLE
Validation with 2 inputs

### DISCUSSION THREAD

**Discussion by teodorfrunza** [postmanlabs/postman-flows#83] [Ideas]
I believe the flows should provide a way to validate the request from a call against another response from another request. For example my first request returns an item with an id and my second request returns a list of items. I want to extract the item from the list that has the id from the first request. As of now, validate accepts only one input and I could not find a workaround, even with merge or concatenate.

**Comment by wmaynard** [comment]
Definitely a major issue for me too.

I think I found a workaround but it's so dirty I don't really recommend it: it's possible to create an endpoint solely to test the input and return a status code based on that, then you can use the status in the validate block.

I agree with you that Validate desperately needs to let us use two inputs.

**Comment by saswatds** [comment]
Hey @teodorfrunza @wmaynard, sorry for the late reply. We were working on implementing an improved version of the validate block which supported complex and nested expression. Now that is done, I will add this feature ASAP.
